templates/overrides/page/page.html.twig

```diff
@@ -60,6 +60,8 @@
       </header>
     {% endif %}
 
+    <a id="main-content" tabindex="-1"></a>
```

Comment:
I've added this hear as it is after the header but not inside the page.content if, because I don't want it to be missing even if there is no content.